elements/howto-menu/demo.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    display: block;
```

Comment:
I wanted to keep the elements as minimal as possible, therefore was not really styling them (I focused on accessibility first). But you're right, if we want to make the elements performant too they should have some visuals too:) I'd say let's do it in next PR, together with animating etc.